src/common/SPELLS/shadowlands/legendaries/warrior.ts

```diff
@@ -77,6 +82,23 @@ const legendaries = {
     icon: 'ability_revendreth_warrior',
     bonusID: 7470,
   },
+  NATURES_FURY: {
+    id: 354161,
+    bonusID: 7471,
+    name: "Nature's Fury",
+    icon: 'ability_ardenweald_warrior',
+  },
+  NATURES_FURY_DAMAGE: {
+    id: 354163,
+    name: "Nature's Fury",
+    icon: 'ability_ardenweald_warrior',
+  },
+  UNITY_WARRIOR: {
```

Comment:
WCL now sends Unity over just as the correct covenant legendary. No need to add unity into the code base